federation/pkg/federation-controller/replicaset/replicasetcontroller.go

```diff
@@ -43,6 +43,7 @@ import (
 	"k8s.io/kubernetes/pkg/util/flowcontrol"
 	"k8s.io/kubernetes/pkg/util/wait"
 	"k8s.io/kubernetes/pkg/util/workqueue"
+	_ "k8s.io/kubernetes/pkg/util/workqueue/prometheus"
```

Comment:
The new github review UI is confusing. I definitely didn't type "Done" to this comment..

Anyway, do you mean importing it in pkg/controller? I'd rather keep this line next to the workqueue import line for easier tracking.